<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPPO</class>
 <widget class="QMainWindow" name="MainPPO">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainPPO</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="1">
     <widget class="QPushButton" name="LoadData">
      <property name="text">
       <string>Load Data</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Number of Patients :</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="analyzeDataButton">
      <property name="text">
       <string>Analyze Data</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="generateDataButton">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Generate Data</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Number of Measurements:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="themeSwitchButton">
      <property name="text">
       <string>Change Theme</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QSpinBox" name="numPatientsSpinBox"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="7" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Patients Data:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QSpinBox" name="numMeasurementsSpinBox"/>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="saveToCSVButton">
      <property name="text">
       <string>Save to CSV</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>497</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Analysis</string>
    </property>
    <addaction name="actionCalculateAverage"/>
    <addaction name="actionCalculateMaxMin"/>
    <addaction name="actionHealthOverview"/>
    <addaction name="actionStandardDeviation"/>
    <addaction name="actionVariance"/>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSet_Default_Directory"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuAnalysis"/>
   <addaction name="menuExit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCalculateAverage">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Average Values</string>
   </property>
  </action>
  <action name="actionCalculateMaxMin">
   <property name="text">
    <string>Max/Min Values</string>
   </property>
  </action>
  <action name="actionHealthOverview">
   <property name="text">
    <string>Health Overview</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSet_Default_Directory">
   <property name="text">
    <string>Set Default Directory</string>
   </property>
  </action>
  <action name="actionStandardDeviation">
   <property name="text">
    <string>Standard Deviation</string>
   </property>
  </action>
  <action name="actionVariance">
   <property name="text">
    <string>Variance</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
